### PR TITLE
added tracing to load_var and set_pipeline.

### PR DESCRIPTION
**Pull request by evanchaoli**
## What does this PR accomplish?

To add tracing to steps `load_var` and `set_pipeline`.

## Changes proposed by this PR:

This is a tiny change, just added tracing to steps `load_var` and `set_pipeline` similar to `get/put/task/check` steps.

## Notes to reviewer:

NA

## Contributor Checklist

- [x] Followed [Code of conduct], [Contributing Guide] & avoided [Anti-patterns]
- [x] [Signed] all commits
- [x] ~Added tests (Unit and/or Integration)~
- [ ] Updated [Documentation]
- [x] Updated [Release notes]

[Code of Conduct]: https://github.com/concourse/concourse/blob/master/CODE_OF_CONDUCT.md
[Contributing Guide]: https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md
[Anti-patterns]: https://github.com/concourse/concourse/wiki/Anti-Patterns
[Signed]: https://help.github.com/en/github/authenticating-to-github/signing-commits
[Documentation]: https://github.com/concourse/docs
[Release notes]: https://github.com/concourse/concourse/tree/master/release-notes

## Reviewer Checklist

- [x] Code reviewed
- [x] ~Tests reviewed~
- [ ] Documentation reviewed
- [x] Release notes reviewed
- [x] PR acceptance performed
- [x] ~New config flags added? Ensure that they are added to the [BOSH](https://github.com/concourse/concourse-bosh-release) and [Helm](https://github.com/concourse/helm) packaging; otherwise, ignored for the [integration tests](https://github.com/concourse/ci/tree/master/tasks/scripts/check-distribution-env) (for example, if they are Garden configs that are not displayed in the `--help` text).~

### CODE REVIEW

**Comment by evanchaoli** [comment]
I tried the feature `tracing` with Jaeger today, and found the `load_var` and `set_pipeline` steps missed tracing spans. To make a build have complete spans, I added spans to `load_var` and `set_pipeline`.

File: release-notes/latest.md [hunk 1]
```diff
@@ -77,3 +77,7 @@ Currently the only API action that can be limited in this way is `ListAllJobs` -
 #### <sub><sup><a name="5222" href="#5222">:link:</a></sup></sub> feature
 
 * Proxy support for NewRelic emitter
+
+#### <sub><sup><a name="5706" href="#5706">:link:</a></sup></sub> fix
+
+* Steps `load_var` and `set_pipeline` missed tracing, @evanchaoli added tracing to these steps. #5706
```
Comment (jamieklassen) [inline release-notes/latest.md:83]:
```suggestion
* @evanchaoli added spans for the `load_var` and `set_pipeline` steps when [distributed tracing](https://concourse-ci.org/tracing.html) is enabled. #5706
```

File: release-notes/latest.md [hunk 2]
```diff
@@ -77,3 +77,7 @@ Currently the only API action that can be limited in this way is `ListAllJobs` -
 #### <sub><sup><a name="5222" href="#5222">:link:</a></sup></sub> feature
 
 * Proxy support for NewRelic emitter
+
+#### <sub><sup><a name="5706" href="#5706">:link:</a></sup></sub> fix
```
Comment (jamieklassen) [inline release-notes/latest.md:81]:
```suggestion
#### <sub><sup><a name="5706" href="#5706">:link:</a></sup></sub> feature
```

**Comment by evanchaoli** [comment]
> could you mention these span operations in the [docs](https://github.com/concourse/docs/blob/master/lit/docs/operation/tracing.lit#L85-L97)?

@jamieklassen  This is the PR https://github.com/concourse/docs/pull/337

**Pull request merged**